mlflow/types/llm.py

```diff
@@ -337,16 +365,7 @@ def __post_init__(self):
         self._convert_dataclass_list("choices", ChatChoice)
         if isinstance(self.usage, dict):
             self.usage = TokenUsageStats(**self.usage)
-        if not isinstance(self.usage, TokenUsageStats):
-            raise ValueError(
-                f"Expected `usage` to be of type TokenUsageStats or dict, got {type(self.usage)}"
-            )
-
-    def __setattr__(self, name, value):
-        # A hack to ensure users cannot overwrite 'object' field
-        if name == "object" and value != "chat.completion":
-            raise ValueError("`object` field must be 'chat.completion'")
-        return super().__setattr__(name, value)
```

Comment:
removing these checks as i received feedback that:
1. it's annoying for token usage to be required
2. it's annoying to not be able to set chat.completion to something else

Reply:
just out of curiosity, what is `something else`? Any example?

Reply:
at the moment i didn't hear any specific example, but we received some feedback saying that it was weird that they couldn't set this. i kind of agree—it makes this field a bit useless. we have the default as `chat.completion`, but if users have some use-case where they want to return something else, then we shouldn't limit them